helpers/helpers/tests/test_firehose.py

```diff
@@ -144,18 +145,28 @@ def test__parse_s3_uri():
     }
 
 
+@contextmanager
```

Comment:
Was this change related to the ticket or something else?

Reply:
basically all unit tests in json_schema ran fine
but if i ran nrlf test unit then one test in test_json_schema failed
because there was a side effect a test in test_firehose.py that changed the current working directory
so i needed to make sure that I changed the working dir back to it’s original state